azurelinuxagent/ga/update.py

```diff
@@ -1017,6 +1023,52 @@ def _ensure_firewall_rules_persisted(dst_ip):
             message=msg,
             log_event=False)
 
+    def _add_accept_tcp_firewall_rule_if_not_enabled(self, dst_ip):
+        # Helper to execute a run command, returns True if no exception
```

Comment:
Can you move this comment inside the  `_execute_run_command` function please. It gave me the false sense that `_add_accept_tcp_firewall_rule_if_not_enabled` was doing this.

Reply:
Addressed